src/main/java/com/minecolonies/coremod/blocks/BlockHutField.java

```diff
@@ -94,15 +98,35 @@
     private void initBlock()
     {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'REGISTRY_NAME' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1192)

Reply:
Apparently this is critical though